comparisons/CompareGraphs.py

```diff
@@ -100,6 +101,8 @@
         hToCompare[iFile].SetDirectory(0)
         hToCompare[iFile].SetStats(0)
         hToCompare[iFile].Scale(scale)
```

Comment:
The `Scale` method should be called once for all, otherwise if you scale first by `scale` and then by the integral, you overwrite the `scale` parameter basically. Alternatively you can just swap the two (first `normalize` and then `scale`)

Reply:
I think that you would never both scale and normalize the histogram. Right?
So the overwrite of the scale effect could be fine. We can also throw an error/warning if you try to use both.

Reply:
In principle, one could do both, and in this case we can add a warning to avoid that one does it unintentionally. If we decide instead that it is not foreseen, I would put an error if `normalize` is `True` and `scale` is `1.` (but I have a slightly preference for the first option, that gives more flexibility)

Reply:
Done